src/front-end/src/helper-methods.js

```diff
@@ -2,6 +2,27 @@ import firebase from './firebase-init';
 
 const auth = firebase.auth();
 
+export function resendVerificationEmail(user) {
```

Comment:
Why make a new method instead of reusing `mySendEmailVerification` and adapting it as needed?